tests/scripts/all.sh

```diff
@@ -2610,6 +2610,22 @@ component_test_tls13_experimental_with_everest () {
     if_build_succeeded tests/ssl-opt.sh
 }
 
+component_test_tls13_experimental_with_rsa () {
+    msg "default config with MBEDTLS_SSL_PROTO_TLS1_3_EXPERIMENTAL enabled, with rsa"
+    scripts/config.py   set MBEDTLS_SSL_PROTO_TLS1_3_EXPERIMENTAL
+    scripts/config.py unset MBEDTLS_SSL_USE_MPS
```

Comment:
```suggestion
```

Reply:
I just remove this component as it has been covered, please see comments: https://github.com/ARMmbed/mbedtls/pull/5127#discussion_r755118046.